ts/session/conversations/ConversationController.ts

```diff
@@ -693,7 +694,7 @@ async function leaveClosedGroup(groupPk: PubkeyType | GroupPubkeyType, fromSyncM
     window?.log?.info(
       `We are leaving the group ${ed25519Str(groupPk)}. Sending our leaving messages.`
     );
-    let failedToSent03LeaveMessage = false;
+    let failedToSent03LeaveMessage = true;
```

Comment:
Just confirmed this is correct and not part of debugging / testing?

Reply:
Never mind, I see you are assuming true unless we fail somehow. Then you set it to false.